Source/XCDBLD/BuildArguments.swift

```diff
@@ -6,10 +6,10 @@ public struct BuildArguments {
 	/// binary.
 	public enum BitcodeGenerationMode: String {
 		/// Only bitcode marker will be embedded.
-		case marker = "marker"
+		case marker
 
 		/// Full bitcode will be embedded.
-		case bitcode = "bitcode"
+		case bitcode
```

Comment:
I've made the change due to the by-default activated rule `redundant_string_enum_value` rule of SwiftLint. So, seems there's enough common sense on the SwiftLint community to enable this by default.